frontend/src/components/ScoreBadge.tsx

```diff
@@ -3,11 +3,24 @@ import classNames from "classnames"
 
 import styles from "./ScoreBadge.module.scss"
 
+function round(num: number) {
+    return Math.round((num + Number.EPSILON) * 100) / 100
```

Comment:
ELI5 pls? In a comment, perhaps?

Reply:
`// force gud rounding https://stackoverflow.com/a/11832950/7290888`

Reply:
update: use toFIxed(2) to convert to string instead of rounding then converting that